src/kibana/directives/saved_object_finder.js

```diff
@@ -67,20 +77,22 @@ define(function (require) {
           if (!$scope.userOnChoose) {
             return hit.url;
           }
+
+          return '#';
```

Comment:
I think this is causing the entire page to reload when searches are added to the dashboard. What is the `'#'` supposed to accomplish?

pr:
![2015-06-15 19_17_56](https://cloud.githubusercontent.com/assets/1329312/8174587/44e4045e-1393-11e5-9124-3cf3d46c882f.gif)

master:
![2015-06-15 19_26_47](https://cloud.githubusercontent.com/assets/1329312/8174680/7fd9b1ac-1394-11e5-82d0-5d069e9f2653.gif)